vmdb/app/models/vm_microsoft.rb

```diff
@@ -7,4 +7,8 @@ def self.calculate_power_state(raw_power_state)
     else                        super
     end
   end
+
+  def supports_migrate?
+    false
+  end
```

Comment:
@fryguy in case TemplateMicrosoft code returns false on the line that i pointed out to @jrafanie in my comment earlier https://github.com/ManageIQ/manageiq/blob/master/vmdb/app/controllers/application_controller.rb#L2159 includes_template? returns true if any of the selected ids from list view are Templates and task being performed is not "clone".
We don't show migrate button on individual Template summary screens.